src/langs/fr.json

```diff
@@ -256,5 +256,22 @@
   "DELETE_MEMBERSHIP_MODAL_CONFIRM_BUTTON": "Confirmer",
   "DELETE_OWN_MEMBERSHIP_MESSAGE": "Êtes-vous sûr de vouloir supprimer votre propre autorisation sur cet élément ? Vous perdrez l'accès à cet élément, cette action est irréversible.",
   "DELETE_MEMBERSHIP_MESSAGE": "Êtes-vous sûr de vouloir supprimer l'accès en {{permissionLevel}} sur cet élément à {{name}} ? Il leur sera impossible d'accéder à cet élément.",
-  "DELETE_LAST_ADMIN_ALERT_MESSAGE": "Vous n'êtes pas autorisé à supprimer le dernier administrateur de cet élément."
+  "DELETE_LAST_ADMIN_ALERT_MESSAGE": "Vous n'êtes pas autorisé à supprimer le dernier administrateur de cet élément.",
+  "CONFIRM_DELETE_SHORT_LINK_TITLE": "Confirmez-vous la suppression du lien rapide ?",
+  "CONFIRM_DELETE_SHORT_LINK_MSG": "Vous êtes sur le point de supprimer le lien rapide {{shortLink}}, voulez-vous continuer ?",
+  "DELETE_BTN": "Supprimer",
+  "SAVE_BTN": "Sauvegarder",
+  "CREATE_SHORT_LINK_TITLE": "Créer un nouveau lien rapide",
+  "EDIT_SHORT_LINK_TITLE": "Modifier le lien rapide",
+  "DELETE_SHORT_LINK_TITLE": "Supprimer le lien rapide",
+  "SHORTEN_LINK_TOOLTIP": "Raccourcir le lien",
+  "ALIAS_VALID_MSG": "L'alias est valide",
+  "ALIAS_CHECKING_MSG": "Vérification de la disponibilité...",
+  "ALIAS_UNCHANGED_MSG": "L'alias est inchangé",
+  "ALIAS_ALREADY_EXIST": "L'alias existe déjà",
+  "GENERATE_ALIAS_TOOLTIP": "Générer un alias",
+  "SHORT_LINK_MIN_CHARS_ERROR": "L'alias doit au moins contenir {{data}} caractères.",
+  "SHORT_LINK_MAX_CHARS_ERROR": "L'alias ne doit pas dépasser les {{data}} caractères.",
+  "SHORT_LINK_INVALID_CHARS_ERROR": "L'alias contient des caractères non valides ({{data}}).",
```

Comment:
Let's have either `lien rapide` or `alias` but not both. Tho I know alias is the column name not sure it's logic for other users.